src/webui/components/Icon/types.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @prettier
+ * @flow
+ */
+
+import { Icons } from './index';
+
+export interface IIconsMap {
+  brazil: string;
+  spain: string;
+  china: string;
+  nicaragua: string;
+  pakistan: string;
+  austria: string;
+  india: string;
+  earth: string;
+  verdaccio: string;
+  [key: string]: string;
```

Comment:
What is this for ? 🤔

Reply:
I am using an index signature..Please check it out : https://flow.org/en/docs/types/objects/ -> _An indexer can be optionally named, for documentation purposes...._ 😉